src/tests/Myr_random.test.js

```diff
@@ -0,0 +1,125 @@
+import Myr from "../myr/Myr";
+
+const myr = new Myr();
+
+describe("Random Function Tests", () => {
+    
+    //randomInt
+    it("randomInt should move box to random x coordinate", () => {
+        let xPos = myr.randomInt();
+        myr.setPosition(xPos, 1, 1);
+        expect(myr.cursor.position).toEqual({ x: xPos, y: 1, z: 1 });
+    });
+
+    it("randomInt value should be within default range", () => {
+        let within = 0, xPos = myr.randomInt();
+        xPos = myr.randomInt();
+        myr.setPosition(1, 1, xPos);
+        if(xPos >= -40 && xPos < 40) {
+            within = 1;
+        }
+        expect(within).toEqual(1);
+    });

```

Comment:
Without changing the overall code, you should not use 1/0 for true/false in JS. You should be using true/false. There is also extra complexity. Read about how expect works [here](https://jestjs.io/docs/en/expect). There are better functions for checking booleans that toEqual.